src/screens/AddCustomer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import BaseHeader from '../components/BaseComponents/BaseHeader';
+import TextField from '../components/TextField';
+import Checkbox from '../components/Checkbox';
+import Button from '../components/Button';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    header: {
+      marginTop: '20px',
+      color: theme.palette.text.primary,
+    },
+    content: {
```

Comment:
content and form margining contradicts each other

`content` styling can probably be moved into `form` (the corresponding wrapper div isn't needed)